package.json

```diff
@@ -157,6 +157,51 @@
             "work_id"
           ]
         }
+      },
+      {
+        "name": "paw_generate_prompt",
+        "displayName": "Generate PAW Prompt",
+        "modelDescription": "Create a customizable prompt file for a PAW agent on demand. The agent must specify: 1) template_key - the base template to use (e.g., '03A-implement', '03C-pr-review'), and 2) filename - the exact output filename (e.g., '03A-implement-phase3.prompt.md'). Valid template keys: 01A-spec, 02A-code-research, 02B-impl-plan, 03A-implement, 03B-review, 03C-pr-review, 03D-review-pr-review, 04-docs, 05-pr, 0X-status. The agent determines the appropriate template and filename based on user intent.",
+        "toolReferenceName": "pawPromptGenerator",
+        "canBeReferencedInPrompt": true,
+        "inputSchema": {
+          "type": "object",
+          "properties": {
+            "work_id": {
+              "type": "string",
+              "description": "The normalized Work ID (feature slug, e.g., 'auth-system')"
+            },
+            "template_key": {
+              "type": "string",
+              "description": "The template key identifying which prompt template to use. This is the base filename without .prompt.md extension.",
+              "enum": [
+                "01A-spec",
+                "02A-code-research",
+                "02B-impl-plan",
+                "03A-implement",
+                "03B-review",
+                "03C-pr-review",
```

Comment:
We've been missing some prompts that we can add now as part of this work.

This is a 02C-planning-pr-review which uses the impl plan agent to review PR comments on the planning PR.
04B-pr-review and 04C-review-pr-review, along with 05B-pr-review and 05C-review-pr-review, which are for responding to the docs PR and the final PR.